src/common/Cargo.toml

```diff
@@ -10,6 +10,9 @@ repository = { workspace = true }
 [package.metadata.cargo-machete]
 ignored = ["workspace-config", "workspace-hack", "task_stats_alloc"]
 
+[package.metadata.cargo-udeps.ignore]
+normal = ["workspace-config", "workspace-hack", "task_stats_alloc"]
```

Comment:
There should be no dependency on `workspace-config` and `task_stats_alloc` for library crates. 🤔 Maybe we only add them to the ignored list of `cmd` and `cmd_all`?

Reply:
Ok, such a lazy boy I am 🫣